hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -1203,7 +1265,9 @@ def _draw_bounded_integer(
         bits = gap.bit_length()
         probe = gap + 1
 
-        if bits > 24 and self._cd.draw_bits(3, forced=None if forced is None else 0):
+        if bits > 24 and self.draw_boolean(
+            7 / 8, forced=None if forced is None else False
+        ):
```

Comment:
Explicitly confirming, we're good to use `draw_boolean()` now?  (iirc yes, but worth checking)

Reply:
Yeah. After https://github.com/HypothesisWorks/hypothesis/commit/9283da35aab9e370ed37620039639b12f5fd326e, `draw_boolean` produces the correct distribution here, at least in my (admittedly not rigorous) testing.